package.json

```
@@ -5,26 +5,32 @@
     "commit": "cz",
     "dev": "MYNE_CHAT_GIT_HASH=$(git rev-parse --short HEAD) next dev -p 8080",
     "build": "MYNE_CHAT_GIT_HASH=$(git rev-parse --short HEAD) next build",
+    "export": "next export",
     "start": "next start -p 8080",
     "lint": "next lint",
     "format": "prettier --write --ignore-path .gitignore .",
     "mocks": "node ./mocks/index.js",
     "test": "jest --watch",
-    "test:ci": "jest --ci"
+    "test:ci": "jest --ci",
+    "test:coverage": "jest --coverage"
   },
   "dependencies": {
+    "@ethersproject/bytes": "^5.5.0",
     "grommet": "^2.19.1",
     "grommet-icons": "^4.7.0",
     "immer": "^9.0.7",
     "lodash": "^4.17.21",
     "next": "12.0.4",
+    "peer-id": "^0.15.4",
```

```suggestion
    "peer-id": "0.16.0",
```
I suggest to use same version of peerId as `core`

---

Sadly this version does the upgrade from `v14` to `v15` which we can't use in Vercel at the moment. See implementation in e06a20289a9ccf0d4779304631388566764ad2ab and explanation in https://github.com/hoprnet/myne-chat/issues/81#issuecomment-1009271002.

---

Solving as per comment.